src/main/resources/static/js/subscription.js

```diff
@@ -722,8 +751,14 @@ jQuery(document).ready(function () {
                     }
                     item[0].requirements[i] = new conditions_model(conditions_array);
                 }
-                for (i = 0; i < item[0].notificationMessageKeyValues.length; i++) {
-                    item[0].notificationMessageKeyValues[i] = new formdata_model(item[0].notificationMessageKeyValues[i])
+                if (item[0].notificationMessageKeyValues.length == 1 && item[0].notificationMessageKeyValues[0].formkey == "") {
```

Comment:
You edit the first element in a list, the list is later sent further, sure one could get the first element from the list, edit it, then put it back in a list. But I believe it is out of the scope for this story since it is only about a small change in this function.

Reply:
Fair enough, but for readability in the logical statement it could atleast be bound to a variable above with a good name that is self explainable.

Reply:
Done.